src/templates/next/components/shared/CustomCssClass.tsx

```diff
@@ -0,0 +1 @@
+export const ComponentContent = "component-content"
```

Comment:
nit: can rename this to a `.ts` file + don't need caps at the start since this is not a React component